components/governance/org.wso2.carbon.governance.generic/src/main/java/org/wso2/carbon/governance/generic/util/GenericArtifactUtil.java

```diff
@@ -56,7 +56,7 @@ public static boolean addRXTResource(String path,String rxtConfig, Registry regi
 
         OMElement element = buildOMElement(rxtConfig);
         if (!CommonUtil.validateXMLConfigOnSchema(
-                RegistryUtils.decodeBytes(rxtConfig.getBytes()), "rxt-ui-config")) {
+                RegistryUtils.decodeBytes(element.toString().getBytes()), "rxt-ui-config")) {
```

Comment:
can this element ever be null?

Reply:
No. This cannot be true since there is a validation to check the null value of `rxtConfig` which is used to create the `element` object.